pkg/convert/testdata/programs/map_keys/pcl/main.pp

```diff
@@ -0,0 +1,18 @@
+resource "aResource" "complex:index/index:resource" {
+  __logicalName = "a_resource"
+  aMapOfBool = {
+    CAPS       = true
+    camelCase  = false
+    snake_case = false
+    PascalCase = true
+  }
+}
+
+output "someMapOutput" {
+  value = {
+    CAPS       = aResource.result
+    camelCase  = 1
+    snakeCase  = 2
```

Comment:
Note that these are still being renamed. Fixing this is a bit more involved. If we just flip the logic to assume things are maps not objects then we end up renaming a lot of `{ k: v}` expressions in TF source code which are then used for object values.

Reply:
It is probably fine that output objects _are_ renamed because the type of these becomes are object type from component outputs, not a map.